samples/unit-tests/navigator/navigator/demo.js

```diff
@@ -1407,3 +1407,56 @@ QUnit.test('Navigator, testing method: getBaseSeriesMin', function (assert) {
         );
     });
 });
+QUnit.test('Navigator updates and redraws', function (assert) {
+    let redrawsAmount = 0;
+    const chart = Highcharts.stockChart('container', {
+        chart: {
+            type: 'line',
+            events: {
+                redraw() {
+                    redrawsAmount++;
+                }
+            }
+        },
+        series: [{
+            type: 'line',
+            data: [1, 2, 3, 4, 5, 6, 7, 8]
+        }, {
+            type: 'line',
+            data: [8, 7, 6, 5, 4, 3, 2, 1]
+        }]
+    });
+    chart.update({
+        navigator: {
+            height: 100,
+            margin: 50,
+            baseSeries: 1,
+            handles: {
+                backgroundColor: 'red',
+                height: 30,
+                width: 30,
+                symbols: ['circle', 'circle']
+            }
+        }
+    });
+    assert.strictEqual(
+        redrawsAmount,
+        1,
+        'Updating multiple navigator options should trigger only one redraw.'
+    );
+    assert.strictEqual(
+        chart.navigator.handles[0].symbols[0],
+        'circle',
+        'Navigator handles should be updated to new symbol.'
+    );
+    assert.strictEqual(
+        chart.navigator.handles[0].height,
+        30,
+        'Navigator handles should be updated to new height.'
+    );
+    assert.strictEqual(
+        chart.navigator.handles[0].width,
+        30,
+        'Navigator handles should be updated to new width.'
+    );
```

Comment:
I see you update more options than the test have asserts. How about adding more of them (height, margin and baseSeries too)? Or do we have this test somewhere already? If yes, then maybe merge the tests first?